public/index.js

```diff
@@ -0,0 +1,15 @@
+"use strict";
+
+var express = require('express');
+
+var db = require('./database/index.js');
+
+var app = express();
+var PORT = 3004;
+app.use(express["static"]('public'));
+app.get("/", function (req, res) {
+  res.send('Welcome to the Details Service');
+});
```

Comment:
static should handle serving GET requests to the / endpoint. Maybe try commenting this out and seeing if you still get the behavior you expect?